client/src/hooks/useWindowDimensions.ts

```diff
@@ -0,0 +1,28 @@
+import { useState, useEffect } from 'react';
+
+type Dimensions = {
+  innerWidth: number
+  innerHeight: number
+}
+
+const useWindowDimensions = (): any => {
+  if (typeof window !== "undefined") {
+    const [windowDimensions, setWindowDimensions] = useState<Dimensions>({
+      innerWidth: window.innerWidth,
+      innerHeight: window.innerHeight
+    });
+
+    useEffect(() => {
+      const handleResize = () => {
+        setWindowDimensions({ innerWidth: window.innerWidth, innerHeight: window.innerHeight });
+      }
+
+      window.addEventListener('resize', handleResize);
+      return () => window.removeEventListener('resize', handleResize);
+    }, []);
+
+    return windowDimensions;
+  }
+}
+
+export default useWindowDimensions;
```

Comment:
Add New Line

Reply:
I added the line in my other PRs, sorry I just noticed the comment. 
Just to avoid conflict when rebasing, I added it in task [SATS-304]